tutorials/3ch.md

```diff
@@ -366,17 +366,17 @@ process. The `ProcessInfo` type it returns contains the local node id and a list
 registered names, monitors and links for the process. The call returns `Nothing` if the
 process in question is not alive.
 
-### Monad Transformer Stacks 
+### Monad Transformer Stacks
```

Comment:
Everything is ok, until you make an instance of `BaseControl` to Process monad. And this is 3rd party package, that I hope mentions possible problems